airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -71,6 +72,14 @@ def __init__(self, start_date: str, conversion_window_days: int, **kwargs):
         self._start_date = start_date
         super().__init__(**kwargs)
 
+    @property
+    def time_zone(self):
+        account = next(Accounts(self.google_ads_client).read_records(sync_mode=None), {})
+        time_zone_name = account.get("customer.time_zone")
+        if time_zone_name:
+            return Timezone(time_zone_name)
+        return "local"
+
```

Comment:
Please add an annotation for returnable values. Can this function return one type any time?

Reply:
the property "time_zone" can be called several times during one sync. Please add some cache mechanism for its value

Reply:
updated, added unit_test